cmd/verify_release/main.go

```diff
@@ -27,23 +27,38 @@ import (
 )
 
 var (
-	publicKeyFile = flag.String("public_key", "", "Path to file containing the public key used to sign the manifest")
+	publicKeyFile = flag.String("public_key", "", "Path to file containing the public key used to sign the manifest. If unset, uses the contents of the RELEASE_PUBLIC_KEY environment variable.")
 	manifest      = flag.String("manifest", "", "Path to the signed manifest")
 )
 
 func main() {
+	var pubkey string
+
 	flag.Parse()
 	if err := validateFlags(); err != nil {
 		glog.Exitf("Invalid flag(s):\n%s", err)
 	}
 
+	if len(*publicKeyFile) > 0 {
+		k, err := os.ReadFile(*publicKeyFile)
+		if err != nil {
+			glog.Exitf("failed to read public key file: %v", err)
+		}
+		pubkey = string(k)
+	} else {
+		pubkey = os.Getenv("RELEASE_PUBLIC_KEY")
```

Comment:
Nit: might be worth popping the env var name in a const and refer to it from Getenv and the fmt.Errorf?

Reply:
Yes, and we should set defaults consistent with https://github.com/f-secure-foundry/armory-drive/blob/ft/Makefile#L93

Reply:
Fixed in ea0a7c5ed78290c1e5cb4de8e082bfd791d5e3f0